src/Syntax.jl

```diff
@@ -134,23 +134,26 @@ Compiles stock and flow syntax of the line-based block form
     symbol_r => flow_name_1(dyvar_k) => symbol_q
     symbol_z => flow_name_2(dyvar_g * param_v) => symbol_p
     ☁       => flow_name_3(symbol_c + dyvar_b) => symbol_r
-    symbol_j => flow_name_4(param_l + symbol_m) => TODO
+    symbol_j => flow_name_4(param_l + symbol_m) => CLOUD
     ...
     symbol_y => flow_name_n(dyvar_f) => ☁
 ```
 into a StockAndFlowF data type for use with the StockFlow.jl modelling system.
 """
 macro stock_and_flow(block)
     Base.remove_linenums!(block)
-    syntax_lines = parse_stock_and_flow_syntax(block.args)
-    saff_args = stock_and_flow_syntax_to_arguments(syntax_lines)
-    return StockAndFlowF(
-        saff_args.stocks,
-        saff_args.params,
-        map(kv -> kv.first => get(kv.second), saff_args.dyvars),
-        saff_args.flows,
-        saff_args.sums,
-    )
+    block_args = block.args
+    return quote
```

Comment:
It turns out there's a good reason to return an expression from macros in regards to error handling. See https://www.patrickfournier.ca/pages/blog/post_throwing-macro-julia/index.html

This fixes the issue I saw before in https://github.com/AlgebraicJulia/StockFlow.jl/pull/80/files#diff-f648c20bf211c2ff365353357788e0fa370300b38a879e4115bb4de79898bec2L260 

Instead of being a `LoadError`, it now throws an exception after the macro returns, and `@test_throws` is usable directly as far as I can tell. E.g. if I remove `@test_throws` I get a sensible, expected error now that is caught by the test framework.

```
Error During Test at StockFlow.jl/test/Syntax.jl:262
  Got exception outside of a @test
  a is not a known stock.
  Stacktrace:
    [1] error(s::String)
      @ Base ./error.jl:35
    [2] assemble_stock_definitions(stocks::Vector{Symbol}, flows::Vector{Tuple{Symbol, Expr, Symbol}}, sum_variables::Vector
{Tuple{Symbol, Vector{Symbol}}})
```

(@neonWhiteout might be of interest to you. I tried to update foot/feet similarly but encountered difficulties.)